<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>N_GCS</class>
 <widget class="QMainWindow" name="N_GCS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>NGroundControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(255, 255, 255)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="WindowStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="startPage">
     <widget class="QWidget" name="logo" native="true">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>340</y>
        <width>251</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: url(:/logo/Nonamed-logo.png);
background-color: transparent;</string>
      </property>
     </widget>
     <widget class="QLabel" name="logo_name">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>580</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>N-GCS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainPage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QStackedWidget" name="SettingStacked">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>240</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Serial">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>64</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>64</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Data Bits</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Flow Control</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>64</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>64</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Stop Bits</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>241</width>
          <height>191</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string> Port setup</string>
        </property>
        <widget class="QComboBox" name="PortComboBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="BaudComboBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="DataComboBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>70</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="FlowComboBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>100</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="ParityComboBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>130</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="StopComboBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>160</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="SerialConnectButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>221</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>groupBox</zorder>
       <zorder>label</zorder>
       <zorder>label_9</zorder>
       <zorder>label_10</zorder>
       <zorder>label_11</zorder>
       <zorder>label_12</zorder>
       <zorder>label_13</zorder>
       <zorder>SerialConnectButton</zorder>
      </widget>
      <widget class="QWidget" name="Values">
       <widget class="QLabel" name="AltitudeLabel">
        <property name="geometry">
         <rect>
          <x>93</x>
          <y>28</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Altitude(m)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="SpeedLabel">
        <property name="geometry">
         <rect>
          <x>65</x>
          <y>100</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ground Speed(m/s)</string>
        </property>
       </widget>
       <widget class="QLabel" name="TimeLabel">
        <property name="geometry">
         <rect>
          <x>85</x>
          <y>165</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Flight Time</string>
        </property>
       </widget>
       <widget class="QLabel" name="Altitude_Label">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
       <widget class="QLabel" name="Speed_Label">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
       <widget class="QLabel" name="Time_Label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>200</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Gain">
       <widget class="QTableWidget" name="RpyTableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>241</width>
          <height>231</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <row>
         <property name="text">
          <string>Roll</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Pitch</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Yaw</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Kp</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ki</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kd</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="SerialSendButton">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>170</y>
          <width>231</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="SerialRequestButton">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>210</y>
          <width>231</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QComboBox" name="SettingComboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>240</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">aligment: AlignHCenter;</string>
      </property>
      <item>
       <property name="text">
        <string>Serial</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Values</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gain</string>
       </property>
      </item>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1941</width>
        <height>991</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: url(:/map/map.PNG);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <zorder>frame</zorder>
     <zorder>SettingStacked</zorder>
     <zorder>SettingComboBox</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image/logo.qrc"/>
  <include location="image/map.qrc"/>
 </resources>
 <connections/>
</ui>
